google-cloud-bigquery/lib/google/cloud/bigquery/load_job.rb

```diff
@@ -751,6 +751,9 @@ def schema= new_schema
           # @param [Symbol] mode The field's mode. The possible values are
           #   `:nullable`, `:required`, and `:repeated`. The default value is
           #   `:nullable`.
+          # @param [Array<String>, String] policy_tags The policy tag list or
+          #   single policy tag for the field. Policy tag identifiers are of
+          #   the form `projects/*/locations/*/taxonomies/*/policyTags/*`.
```

Comment:
May want to include the gotcha from the [rest reference](https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#TableSchema): `At most 1 policy tag is currently allowed.`

Reply:
Will do, thanks for the tip.